src/util/BaseLoginController.js

```diff
@@ -92,8 +92,14 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+    // Event triggered at after a page is rendered along with the classname to identify the page
+    postRender: function() {
```

Comment:
We shouldn't do this in postRender - we generally don't modify the hook functions in base classes. I.e. if we do this, then we have to modify every controller that has a postRender hook to makes sure to call the parent class postRender method (i.e. RecoveryChallengeController.js).